scripts/run-dependencies.sh

```diff
@@ -11,6 +11,7 @@ docker-compose pull
 docker-compose --project-name montagu up -d
 
 # Install orderly-web
+pip3 install constellation
```

Comment:
unrelated change?

Reply:
oh yeah, python packaging being crap, shouldn't need this but seem to